allauth/templates/account/email/base_notification.txt

```diff
@@ -0,0 +1,24 @@
+{% extends "account/email/base_message.txt" %}
+{% load i18n %}
+{% load account %}
+{% user_display user as user_display %}
+
+{% block content %}
+Hello {{ user_display }},
+
+We wanted to let you know that there has been a recent change to your account information.
```

Comment:
```suggestion
You are receiving this mail because the following change was made to your account:
```

I would make it a bi more concise ("recent change" / "we wanted to let you know" does not convey much info).